### DISCUSSION TITLE
Broken link on documentation homepage

### DISCUSSION THREAD

**Discussion by NicolasRichel** [encode/django-rest-framework#9117] [Potential Issue]
Hi everyone.

I just landed on the documentation homepage to start learning Django REST Framework.
By browsing across different links on the page I noticed the one about [Web Browsable API](https://restframework.herokuapp.com/) seems to be broken. More specifically it looks like an Heroku app encounter an error.
This is the first link in the list of reasons why you might use Django REST Framework.
So I wanted to bring that to your attention.

**Comment by auvipy** [comment]
I thin the heroku server subscription expired
